<ui version="4.0" >
 <class>mainTabWidget</class>
 <widget class="QTabWidget" name="mainTabWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>408</height>
   </rect>
  </property>
  <property name="currentIndex" >
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>244</width>
     <height>364</height>
    </rect>
   </property>
   <attribute name="title" >
    <string>Insert</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2" >
    <item row="0" column="0" colspan="2" >
     <widget class="QGroupBox" name="groupElements" >
      <property name="title" >
       <string>Elements</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="buttonFence" >
         <property name="text" >
          <string/>
         </property>
         <property name="popupMode" >
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonScript" >
         <property name="text" >
          <string/>
         </property>
         <property name="popupMode" >
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonRoot" >
         <property name="text" >
          <string/>
         </property>
         <property name="popupMode" >
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonFraction" >
         <property name="text" >
          <string/>
         </property>
         <property name="popupMode" >
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonTable" >
         <property name="text" >
          <string/>
         </property>
         <property name="popupMode" >
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" >
     <widget class="QGroupBox" name="groupSymbols" >
      <property name="title" >
       <string>Symbols</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QComboBox" name="comboSymbolTyp" />
       </item>
       <item row="1" column="0" >
        <widget class="QListWidget" name="listSymbols" />
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QPushButton" name="buttonLoad" >
      <property name="text" >
       <string>Load...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QPushButton" name="buttonSave" >
      <property name="text" >
       <string>Save...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>244</width>
     <height>364</height>
    </rect>
   </property>
   <attribute name="title" >
    <string>Edit</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_5" >
    <item row="0" column="0" colspan="2" >
     <widget class="QGroupBox" name="groupGeneralOptions" >
      <property name="title" >
       <string>General options</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <widget class="QFontComboBox" name="fontComboBox" >
       <property name="geometry" >
        <rect>
         <x>80</x>
         <y>70</y>
         <width>151</width>
         <height>34</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>70</y>
         <width>41</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>Font:</string>
       </property>
       <property name="buddy" >
        <cstring>fontComboBox</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="labelColors" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>Colors:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" >
     <widget class="QGroupBox" name="groupSpecificOptions" >
      <property name="title" >
       <string>Specific options</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item rowspan="2" row="0" column="0" >
        <widget class="QStackedWidget" name="stackedWidget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="Table" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>218</width>
            <height>108</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Rows:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>Columns:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>248</width>
            <height>259</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" >
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>104</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" >
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>View MathML</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
